src/platforms/alexa/AlexaEvent.ts

```diff
@@ -30,7 +30,9 @@ import { Context as AWSLambdaContext } from "aws-lambda";
 import { Context as AzureContext } from "azure-functions-ts-essentials";
 import { LambdaLogOptions } from "lambda-log";
 import * as _ from "lodash";
-import { IVoxaIntent, VoxaEvent } from "../../VoxaEvent";
+import * as rp from "request-promise";
```

Comment:
i just realized recently that the correct way to import rp with typescript is `import rp = require('request-promise');`

at least according to https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/request-promise/request-promise-tests.ts#L1

Let's update it in this file since we're already working on this code